README.md

```diff
@@ -110,6 +110,9 @@ distribution regardless its version.
 Packages hosted by [Packagecloud](https://packagecloud.io/gbt/release):
 
 ```shell
+# When you get message "Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8))."
+# use command:
+# curl -L https://packagecloud.io/gbt/release/gpgkey | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/gbt.gpg >/dev/null
 curl -L https://packagecloud.io/gbt/release/gpgkey | sudo apt-key add -
```

Comment:
Could we do that for any Ubuntu distro version by default?

```suggestion
curl -L https://packagecloud.io/gbt/release/gpgkey | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/gbt.gpg >/dev/null
```

Reply:
I'm not sure - I have this problem only on latest LTS 22.04.

Reply:
I would prefer to have one solution only that will just work across all Ubuntu releases. Please could you use the above across few latest Ubuntu versions (14, 16, 18, 20, 22)?

Reply:
Hi, I try check this with docker ubuntu images, but this is probably not good idea, because this are striped out. I don't get any warning in ubuntu:22.04.

But I read deprecation information in man page for apt-key:

![image](https://user-images.githubusercontent.com/326530/219060038-c5b242c4-b27f-4bf5-803f-40f059a751f0.png)

Long story short - probably is safe use old command in Ubuntu <= 22.04 but newer version should use gpg. I understand, have one solution is better, but Ubuntu do not make this easy :-), especially when user (like me) hate warning when use commands :-).